cli/dev.ts

```diff
@@ -9,17 +9,21 @@ Usage:
 if the <dir> is empty, the current directory will be used.
 
 Options:
+    -h, --host       A hos to start the Aleph.js app, default is localhost
```

Comment:
Should be `-hn, --host`. Also typo with `A host`. See [cli/start.ts#L11](https://github.com/alephjs/aleph.js/blob/master/cli/start.ts#L11)